.github/scripts/core_checker.py

```diff
@@ -258,7 +258,7 @@
 
 FREERTOS_IGNORED_PATTERNS = [
     r'.*\.git.*',
-    r'.*mbedtls_config\.h.*',
+    r'.*mbedtls_config*',
```

Comment:
Change how we exclude the mbedtls_config file from this check to ignore the pattern of the file VS calling it out explicitly.